--- kdesdk-4.3.4/umbrello/umbrello/CMakeLists.txt.strictalias	2009-04-05 22:00:34.000000000 +0200
+++ kdesdk-4.3.4/umbrello/umbrello/CMakeLists.txt	2014-04-03 13:21:44.918888367 +0200
@@ -2,7 +2,8 @@ add_subdirectory( pics )
 add_subdirectory( headings )
 add_subdirectory( cmds )
 
-add_definitions (-DQT3_SUPPORT_WARNINGS)
+add_definitions (-DQT3_SUPPORT_WARNINGS
+                 -fno-strict-aliasing)
 
 # Some Boost require exceptions
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}  ${KDE4_ENABLE_EXCEPTIONS}")
